src/node/node_state.h

```diff
@@ -1072,6 +1073,10 @@ namespace ccf
         setup_one_off_secret_hook();
         auto tx = network.tables->create_tx();
 
+        // Clear recovery shares that were submitted to initiate the recovery
+        // procedure
+        share_manager.clear_submitted_recovery_shares(tx);
```

Comment:
We used to clear the shares in the `POST /gov/recovery_share` endpoint, once the recovery was initiated. This wasn't ideal as it meant the encrypted recovery share was never recorded in the ledger. Instead, we now clear the submitted recovery shares once the private recovery is complete.